src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp

```diff
@@ -502,7 +502,7 @@
 
                     item.lastVerifiedDate = data.lastVerifiedDate;
                     if (item.lastVerifiedDate != null && item.lastModifiedDate > item.lastVerifiedDate) {
-                        item.value += '&nbsp; <img src="../images/changed_since_verified.png" width="16">';
+                        item.value += '&nbsp; <img src="../images/changed_since_verified.png" width="16" title="Value changed since last verified">';
```

Comment:
can this be read from words.properties instead of hardcoding it here? I see that there are other titles on this jsp that are also hardcoded. Please update those as well.

Reply:
@svadla-oc OK PR updated, all titles are now read from words.properties